superset-frontend/src/views/components/MenuRight.tsx

```diff
@@ -63,6 +79,9 @@ const StyledAnchor = styled.a`
 
 const { SubMenu } = Menu;
 
+// stole this from findPermission.ts
+const ADMIN_ROLE_NAME = 'admin';
```

Comment:
do you want to export this from `findPermission.ts` so that if it changes there, it will change here, too?